docs/configuration/translation-strings.md

```diff
@@ -23,29 +11,26 @@ You can also translate [Collection Content]() and [Field Names]().
 </video>
 
 When configuring Fields as well as Presets and Bookmarks, you may wish to add additional notes or descriptions to
-clarify their purpose or context.
+clarify their purpose or context. To create and assign a Translation String, follow these steps.
 
-**Create and Manage Translation Strings**\
+**Create and Manage Translation Strings**
 
 1. Navigate to **Settings > Translation Strings**.
 2. Click <span mi btn>add</span> in the page header and a side menu will appear.
-3. Add a key and click **"Create New"**. Another side menu will appear.
+3. Add a key and click **"Create New"** and another side menu will appear.
 4. Select the language and add the corresponding translation.
 5. Click <span mi btn>check</span> to confirm and add the translation.
 6. Repeat steps 3-5 as desired.
 7. Optional:
    - Click and drag <span mi icon>drag_handle</span> to reposition a translation.
    - Click <span mi icon>close</span> to delete a translation.
 8. Click <span mi btn>check</span> to save the translation(s).
-9. Optional: Delete Translation String\
-   Click the translation string to open the side menu, click <span mi btn dngr>delete</span>, then click **Delete**.
 
-Now you can add it to Translation String Inputs in the **Settings Module**, noted with <span mi icon>translate</span> on
-them.
+In the Settings Module, you can assign the Translation String to any inputs with a <span mi icon>translate</span> icon.
 
-**Assign a Translation String**\
+**Assign a Translation String**
 
-1. Navigate to the desired Translation String Input within Settings.\
+1. Navigate to the desired Translation String Input within Settings.
 2. Click <span mi icon>translate</span> and a dropdown menu with all Translation Strings will appear.
```

Comment:
> does this interface allow for mixed values, like 14 $(key-here)? 

Currently it does not, as typical translation strings (throughout the App) are parsed with the notion that they starts with `$t:` with underscore-d key, eg `$t:this_is_my_input_note`.

> Is there value in adding that raw syntax so devs understand it?

Hmm I'm not sure, but one scenario off the top of my head is it might be helpful if they're using them in custom extensions? 🤔

Choosing the keys from the dropdown will input e.g. `$t:boolean_yes` and turn into the chip/pill like display templates. Technically it is indeed possible to manually type it, as seen here:

![chrome_tvOxMc5kcN](https://user-images.githubusercontent.com/42867097/173955099-f9ea31a5-1c0a-4f99-b2c9-77e0b8e7f8ea.gif)